app/controllers/companies_controller.rb

```diff
@@ -15,6 +15,10 @@ def show
     @company = CompanyPresenter.new(company)
   end
 
+  def show_company_shareholders
+    @company = Company.find(params[:id])
```

Comment:
maybe you could just show them in `show` view instead of adding a separate page?

Reply:
I'm not sure what do you mean, because the `show` view is for company view, `shareholders` have different attributes.
You mean to add shareholders to the `show` (company) view ? or set parameter and check in the `show` view, then render `shareholders` instead of `company` view? I mean, I like to see company shareholders on page with no other attributes, I mean for now. I could use show view and just check what to show (company or shareholders of company), this could be good idea because later it will be easily changeable if I like to see company with shareholders on one page.

Reply:
I thought about it and I'm sure that shareholders view will be on separated page, I think it is better way, I like it to be simple and do not give user too much information at one time, specially here, the data are grouped together but I think shareholders should be separated. That is for now :)